docs/pages/deploy-a-cluster/helm-deployments/aws.mdx

```diff
@@ -19,6 +19,19 @@ cluster to Teleport.
 
 (!docs/pages/kubernetes-access/helm/includes/teleport-cluster-prereqs.mdx!)
 
+## Choose a Kubernetes namespace and Helm release name
+
+<Admonition type="note">
```

Comment:
I think it would make sense to take these instructions out of the Admonition box. Since there's not a lot of content in the Prerequisites section, they're pretty noticeable as-is.

Reply:
Still respectfully disagree. The same prerequisites section appears on every page and so people don't tend to pay any attention to it!